db/models/user.js

```diff
@@ -1,12 +1,12 @@
 const mongoose = require("mongoose");
+const Schema = require('mongoose').Schema
```

Comment:
This can be changed to `const Schema = mongoose.Schema;`

Reply:
changes done as required

Reply:
I'm happy that this has been addressed